api/utils/validate-draft-js-input.js

```diff
@@ -0,0 +1,18 @@
+export const validateRawContentState = (input: any) => {
+  if (!input.blocks || !Array.isArray(input.blocks) || !input.entityMap) {
+    trackQueue.add({
+      userId: user.id,
+      event: eventFailed,
+      properties: {
+        reason: 'invalid draftjs data',
+        message,
+      },
+    });
+
+    throw new UserError(
+      'Please provide serialized raw DraftJS content state as content.body'
```

Comment:
Should we have a slightly different error message that's a bit more user-friendly, like "The chat input only accepts inline markdown and code blocks. It looks like you tried to post something else!"

Reply:
Invalid block types (e.g. headings) will just be converted to text. Users will never see this after this PR, only folks who try to send invalid DraftJS content on purpose, so this is fine.